src/lib/remark-plugins/rehype-sanitize/__tests__/rehype-sanitize.test.ts

```diff
@@ -0,0 +1,119 @@
+import type { Root, RootContent, Node } from 'hast'
+import { serialize } from 'next-mdx-remote/serialize'
+import rehypeSanitize, { schema } from '../'
+
+/**
+ * Note: `rehype-sanitize` includes significant test coverage
+ * through `hast-util-sanitize`. Rather than replicate those tests,
+ * this test file includes only a few test cases, to confirm our `schema` is valid.
+ * https://github.com/syntax-tree/hast-util-sanitize/blob/main/test.js
+ *
+ * It's also worth noting we're usually ultimately rendering our markdown
+ * with MDX, rather than going to HTML. So, we'd likely have a whole other
+ * set of concerns related to `jsx` nodes... but thankfully `rehype-sanitize`
+ * strips these by default, which works for use (for now). We also want to
+ * test to confirm that `jsx` nodes are removed, as for now, without better
+ * tooling to sanitize `jsx` nodes, we have to assume that any use of `jsx`
+ * nodes could be malicious.
+ */
+
+describe('rehypeSanitize', () => {
```

Comment:
Tried to be thorough on tests here, while not replicating coverage we already have from [`hast-util-sanitize` tests](https://github.com/syntax-tree/hast-util-sanitize/blob/main/test.js) (which is under the hood of `rehype-sanitize`).